src/mvPlotting.cpp

```diff
@@ -2655,6 +3287,15 @@ DearPyGui::set_configuration(PyObject* inDict, mvCustomSeriesConfig& outConfig)
 	if (PyObject* item = PyDict_GetItemString(inDict, "y2")) { (*outConfig.value)[3] = ToDoubleVect(item); }
 	if (PyObject* item = PyDict_GetItemString(inDict, "y3")) { (*outConfig.value)[4] = ToDoubleVect(item); }
 	if (PyObject* item = PyDict_GetItemString(inDict, "tooltip")) { outConfig.tooltip = ToBool(item); }
+
+	// helper for bit flipping
+	auto flagop = [inDict](const char* keyword, int flag, int& flags)
+	{
+		if (PyObject* item = PyDict_GetItemString(inDict, keyword)) ToBool(item) ? flags |= flag : flags &= ~flag;
+	};
+
+	// flags
+	flagop("no_fit", ImPlotItemFlags_NoFit, outConfig.flags);
```

Comment:
Now it works! In both of the cases of your script it'll fit the plot but that's just because of the line series... if you remove it you'll see the right behaviour

Reply:
Yeah, it should auto-fit but to different ranges: one plot should display the sine wave fitted into the plot area, and the other should display the same wave with a space around it.

Reply:
Yep, that's exactly the behaviour